### ISSUE TITLE
Question regarding the training configs

### ISSUE THREAD

**Issue by YunzeMan** [bradyz/cross_view_transformers#6]
Hi @bradyz, thanks for your beautiful work and for releasing the codebase!

I see in `config/experiment` you only provide the training config for vehicle category. I was wondering what's the right way to change it into the config for road category. Is there anything else to change (e.g. `bev` or `center` keys) other than changing the `override /data` from vehicle to road?

Thanks a lot!

**Comment by bradyz** [comment]
that's correct - the config is now added at [`config/experiment/cvt_nuscenes_road.yaml`](https://github.com/bradyz/cross_view_transformers/blob/master/config/experiment/cvt_nuscenes_road.yaml)
